packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -5,20 +5,44 @@
 import 'dart:async';
 import 'dart:math';
 
+import 'package:collection/collection.dart';
+import 'package:meta/meta.dart';
+
 import '../utils.dart';
 
 // For documentation, see the Chrome "Trace Event Format" document:
 // https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
 
-// Switch this flag to true to dump timeline events to console.
-bool _debugEventTrace = false;
+// Switch this flag to true collect debug info from the timeline protocol. This
+// will also add a button to the timeline page that will download files with
+// this info on click.
+bool debugTimeline = false;
+
+/// Strings that we will build and output to text files for debug purposes.
+///
+/// [debugTraceEvents]: trace events in the order we receive them
```

Comment:
Match the dart doc comment format. You should have a comment above each field instead of one comment for multiple fields.